actions/kexec/v1/cmd/kexec.go

```diff
@@ -0,0 +1,77 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"syscall"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"golang.org/x/sys/unix"
+)
+
+// Release - this struct contains the release information populated when building kexec
+var Release struct {
+	Version string
+	Build   string
+}
+
+const mountAction = "/mountAction"
+
+var kexecCmd = &cobra.Command{
+	Use:   "kexec",
+	Short: "This is a tool for managing storage for bare-metal servers",
```

Comment:
Is this the right description?? I expect some dexcription about kernel exec'ing ???

Reply:
Ah.. this is what happens when you copy from another action! (good spot 👀)